src/main/java/org/cloudfoundry/autosleep/servicebroker/service/AutoSleepServiceInstanceService.java

```diff
@@ -38,33 +39,41 @@
 
     private PasswordEncoder passwordEncoder;
 
+    private Deployment deployment;
+
 
     @Autowired
     public AutoSleepServiceInstanceService(ApplicationRepository appRepository,
                                            ServiceRepository serviceRepository,
                                            GlobalWatcher watcher,
-                                           PasswordEncoder passwordEncoder) {
+                                           PasswordEncoder passwordEncoder,
+                                           Deployment deployment) {
         this.appRepository = appRepository;
         this.serviceRepository = serviceRepository;
         this.watcher = watcher;
         this.passwordEncoder = passwordEncoder;
+        this.deployment = deployment;
     }
 
     @Override
     public ServiceInstance createServiceInstance(CreateServiceInstanceRequest request) throws
             ServiceInstanceExistsException, ServiceBrokerException {
-        log.debug("createServiceInstance - {}", request.getServiceInstanceId());
+        String serviceId = request.getServiceInstanceId();
+        log.debug("createServiceInstance - {}", serviceId);
 
-        AutosleepServiceInstance serviceInstance = serviceRepository.findOne(request.getServiceInstanceId());
+        AutosleepServiceInstance serviceInstance = serviceRepository.findOne(serviceId);
         if (serviceInstance != null) {
             throw new ServiceInstanceExistsException(serviceInstance);
         } else {
             request.setParameters(processParameters(null, request.getParameters()));
             serviceInstance = new AutosleepServiceInstance(request);
+            String baseDashUrl = deployment == null ? "localhost:8080" : deployment.getFirstUri();
```

Comment:
This was in order to return valid a url when running localy, but we can remove it if you like. By the way, I was tempted to add the `http://`prefix to it. What do you think?